package.json

```diff
@@ -23,6 +23,7 @@
   },
   "devDependencies": {
     "@tsconfig/node12": "^1.0.9",
+    "@types/redis-parser": "^3.0.0",
```

Comment:
Note: I took the liberty to take the types from DefinitelyTyped, and remove the local ones. I figured maybe the local types were added because the DefinitelyTyped package didn't exist yet or it was inaccurate

But if you want me to revert this change, let me know! It will work either way

Reply:
Using the DefinitelyTyped version is a good idea, I haven't noticed it exists... Just please move it into ./packages/client/package.json (and not in the root package.json)